### ISSUE TITLE
Tune figures

### ISSUE THREAD

**Issue by huddlej** [blab/cartography#37]
Considerations for Altair figures:

 - [x] Remove grid lines from individual panels
 - [x] Plot the divergence tree instead of the time tree (since genetic distance is what we care about most and not time), also push new Docker image
 - [x] Draw branches on tree panels to clearly communicate that those panels reflect phylogenies and not just a scatterplot that happens to look like a tree
 - [ ] Plot [the "stress" value for each MDS embedding](https://scikit-learn.org/stable/modules/generated/sklearn.manifold.MDS.html) as text on the panel or as text on the x-axis title

Considerations for Euclidean vs. genetic distance figures:

- [x] Remove bootstrap-base std dev (sample counts are so high for these analyses, that bootstrapping doesn't reveal any meaningful variation)
- [x] Fit linear model between genetic and Euclidean distance and plot the corresponding slope and intercept values on each panel as text alongside the R^2 values.

For the H3N2 HA/NA analysis:

 - [x] Figure out a better figure layout than the current 5-row implementation, since this figure runs off the page now
   - [x] Move current 5-row figure (`flu-2016-2018-ha-na-embeddings-by-mcc.png`) to the supplement and rename to `flu-2016-2018-ha-only-vs-ha-na-embeddings-by-mcc.png`
   - [x] Create a 5-panel main figure (using the same name as the original main figure: `flu-2016-2018-ha-na-embeddings-by-mcc.png`) with tree and PCA, MDS, t-SNE, and UMAP embeddings from HA/NA sequences and VI distances annotated in the titles from both the HA-only and HA/NA clusters
 - [x] Assign MCCs colors by first sorting MCCs in descending order of total samples, assigning colors from the standard color schemes TSV, and then sorting by MCC name again for clearer figure legends in Auspice

For late SARS-CoV-2 analysis:

 - [x] Remove the reference/root sequence (Wuhan-1/2019) from the tree after TreeTime step, to avoid large gap in Altair plots for the tree

**Comment by nandsra21** [comment]
Question: would it be best to add stress as possible metadata from the pathogen-embed command so people can use it / any other suggestions that would work?

**Comment by nandsra21** [comment]
leave annotated_embeddings at the tip level, table.tsv will have internal nodes for plotting the divergence tree. Add another parameter for the dataframe table.tsv (add true/false column for internal node, add a filter where necessary)

**Comment by nandsra21** [comment]
root_and_prune_tree (under ha/na scripts) - use for removing strain

**Comment by nandsra21** [comment]
filter by size, order, color by MCC after

**Comment by huddlej** [comment]
Although we can generate stress values for MDS, I don't think including stress in the figures is a blocker to submission, so I'm closing this issue.
